src/main/java/edu/cornell/kfs/kim/bo/impl/CuKimAttributes.java

```diff
@@ -0,0 +1,30 @@
+package edu.cornell.kfs.kim.bo.impl;
+
+import org.kuali.kfs.coa.businessobject.FundGroup;
+import org.kuali.kfs.kim.bo.impl.KimAttributes;
+
+public class CuKimAttributes extends KimAttributes {
+    
+    public static final String FUND_GROUP_CODE = "fundGroupCode";
+    
+    protected String fundGroupCode;
+    
+    protected FundGroup fundGroup;
```

Comment:
Sorry for the confusion.  I was suggesting that since KimAttributes only have codes and no BOs, maybe CuKimAttributes shouldn't have the BO fundGroup, and just use fundGroupCode

Reply:
org.kuali.kfs.kim.bo.impl.KimAttributes does have the related BOs. Check the subFundGroup on line 133.